src/Uno.UI/UI/Xaml/Controls/ItemsStackPanel/ItemsStackPanel.NotImplemented.cs

```diff
@@ -1,9 +1,9 @@
-#if !XAMARIN_IOS && !XAMARIN_ANDROID
+﻿#if !XAMARIN_IOS && !XAMARIN_ANDROID && !NETSTANDARD2_0
```

Comment:
`__WASM__` ?

Reply:
@carldebilly @MatFillion Turned out to be easier to make this change for the whole solution rather than just the files I added, I'll submit a separate PR after this one passes.

Reply:
Agreed.